server/utils/deleteAws.js

```diff
@@ -0,0 +1,45 @@
+const asyncHandler = require("express-async-handler");
+
+exports.deleteImage = asyncHandler(async (Key) => {
+    try {
+        s3.deleteObject({
+            Bucket: process.env.S3_BUCKET,
+            Key
+        }, function (err, data) {
+            if (err) {
+                throw err
+            } else {
+                console.log(data)
+                console.log("File has been deleted successfully");
```

Comment:
Rather than logging the message to the console, maybe we can send a response to the FE on success?

Reply:
I should send a response to the front end inside the helper function? Tried talking this out loud and I am unsure how to send a response in this spot with no access to res. Pass res to the function?

Reply:
Good question! As opposed to passing the req/res to the helper function, maybe have the helper function return a value and based on that value we can send a response in our main route handler. Does that make sense? :)

Reply:
pretty sure it does, thank you for clarifying!